src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -1385,7 +1384,7 @@ internal void SaveAs(string path)
         {
             try
             {
-                Model.CurrentWorkspace.SaveAs(path, EngineController.LiveRunnerRuntimeCore);
+                model.SaveAs(path);
```

Comment:
hmm - I think workspace should be a parameter to this call as well, I believe it can only make it clearer. Saving a DynamoViewModel is not actually what we're doing right? We're saving a workspace.